src/pages/new/volunteer/index.js

```diff
@@ -55,6 +57,7 @@ export default () => (
         },
       ]}
     />
+    <Spacer heightDesktop="100px" heightMobile="80px" />
```

Comment:
Shouldn't there be 120px of spacing in desktop here and 0 in mobile?

Reply:
ok